x/farming/legacy/v2/store_test.go

```diff
@@ -0,0 +1,64 @@
+package v2_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/testutil"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/tendermint/crypto"
+
+	chain "github.com/cosmosquad-labs/squad/app"
+	v1farming "github.com/cosmosquad-labs/squad/x/farming/legacy/v1"
+	v2farming "github.com/cosmosquad-labs/squad/x/farming/legacy/v2"
+	"github.com/cosmosquad-labs/squad/x/farming/types"
+)
+
+func TestMigrateQueuedStaking(t *testing.T) {
+	now := time.Date(2022, 1, 1, 9, 0, 0, 0, time.UTC) // "2022-01-01T09:00:00Z"
+
+	encCfg := chain.MakeTestEncodingConfig()
+	storeKey := sdk.NewKVStoreKey(v1farming.ModuleName)
+	ctx := testutil.DefaultContext(storeKey, sdk.NewTransientStoreKey("transient_test"))
+	ctx = ctx.WithBlockTime(now)
+	store := ctx.KVStore(storeKey)
+
+	stakingCoinDenom := "denom1"
+	farmerAcc := sdk.AccAddress(crypto.AddressHash([]byte("farmer")))
+
+	queuedStaking := types.QueuedStaking{Amount: sdk.NewInt(1000000)}
+	bz := encCfg.Marshaler.MustMarshal(&queuedStaking)
+
+	oldKey := v1farming.GetQueuedStakingKey(stakingCoinDenom, farmerAcc)
+	store.Set(oldKey, bz)
+
+	const currentEpochDays = 2
```

Comment:
Is there a reason why I chose 2 instead of default 1?

Reply:
It's to test if CurrentEpochDays parameter works well and to sure that I didn't hardcoded epoch days as 1.

Reply:
then, the actual migration environment will be 1, so it would be good to add a test case for 1.